### ISSUE TITLE
V1 Pre2P (i.e. PreP2P) Module

### ISSUE THREAD

**Issue by Olshansk** [pokt-network/pocket#32]
# Objective
A stop-gap implementation of the P2P module to unblock the implementation of Consensus & Utility in a localnet environment.

# Origin Document
Add a Pre2P (i.e. PreP2P) implementation to mock allow for sending and broadcasting messages between Pocket network actors and validator nodes. This is not a mock per say, but uses Goland's native networking modules to allow the broadcasting and sending messages to other actors in the Pocket Network. Though it is a full implementation of P2P communication, it is not optimized for large networks, nor does it follow the P2P spec in any way.

In short, it'll be a port of the code that was used in [consensus-pre-prototype](https://github.com/pokt-network/consensus-pre-prototype) to unblock work in the interim and allow for a very simple substitute for P2P in a localnet.

**Creator**: @Olshansk 
**Co-Owners**: @andrewnguyen22 @derrandz 
**Deliverables**: 
- Pre2P module implementation
- Documentation on how to use, build and test
